understanding/audio-only-and-video-only-prerecorded.html

```diff
@@ -121,135 +113,104 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Audio and video-only content can be understood by more people.</dd>
+                  <dt>目標</dt>
+                  <dd>音声のみ及び映像のみコンテンツを、より多くの人々が理解できる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide an alternative when content is perceivable with only one sense.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>コンテンツが一つの感覚でしか知覚できないとき、代替コンテンツを提供する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who can’t fully see or hear content can understand it.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>コンテンツを見たり聞いたりすることが完全にはできない人々が、内容を理解できる。</dd> 
```

Comment:
これで問題ないのですが、他の達成基準の翻訳では People を「人々」ではなく「人」としているので、揃えておきたいです。

```suggestion
                  <dd>コンテンツを見たり聞いたりすることが完全にはできない人が、内容を理解できる。</dd> 
```